server/index.js

```diff
@@ -30,6 +30,14 @@ const getUser = async req => {
 const server = new ApolloServer({
   typeDefs: schemas,
   resolvers,
+  engine: {
+    rewriteError (err) {
+      // logging here
```

Comment:
I suggest to use simple logger like [loglevel](github.com/pimterry/loglevel) here and all around server side